packages/editor/src/components/post-saved-state/style.scss

```diff
@@ -3,7 +3,7 @@
 	align-items: center;
 	width: $icon-button-size - 8px;
 	padding: #{ $grid-size-small * 3 } $grid-size-small;
-	color: $light-gray-900; // Doesn't need to meet AA because button is disabled and it's supporting text.
+	color: $dark-gray-500;
```

Comment:
You could use `$dark-gray-300` if you like. That also meets AA contrast.

Reply:
Thanks — I just tried this out, but I think we should stick with `$dark-gray-500` for now. `$dark-gray-300` adds another gray shade to the toolbar, and I think the hierarchy is more straightforward if we keep the color palette limited there.